product_docs/docs/hadoop_data_adapter/2/05_installing_the_hadoop_data_adapter/index.mdx

```diff
@@ -17,8 +17,10 @@ For platform-specific install instructions, see:
   - [CentOS 7](x86_amd64/04_hadoop_centos7_x86)
   - [SLES 15](x86_amd64/05_hadoop_sles15_x86)
   - [SLES 12](x86_amd64/07_hadoop_sles12_x86)
-  - [Ubuntu 20.04 or 18.04/Debian 10](x86_amd64/09_hadoop__ubuntu18_20_deb10_x86)
-  - [Debian 9](x86_amd64/11_hadoop__deb9_x86)
+  - [Ubuntu 20.04](x86_amd64/09_hadoop_ubuntu20_x86)
+  - [Ubuntu 18.04](x86_amd64/09a_hadoop_ubuntu18_x86)
+  - [Debian 10](x86_amd64/09b_hadoop_deb10_x86)
+  - [Debian 9](x86_amd64/11_hadoop_deb9_x86)
```

Comment:
For some reason (probably my mistake some months ago, the file names have double underscores. Because of that, the link in this file are not resolving. Either the file name should be corrected or the links should be.

Reply:
Made the changes!